wandb/sdk/data_types/image.py

```diff
@@ -508,29 +560,30 @@ def guess_mode(
             )
 
     @classmethod
-    def to_uint8(cls, data: "np.ndarray") -> "np.ndarray":
-        """Convert image data to uint8.
-
-        Convert floating point image on the range [0,1] and integer images on the range
-        [0,255] to uint8, clipping if necessary.
-        """
+    def normalize(cls, data: "np.ndarray") -> "np.ndarray":
```

Comment:
Should be prefixed by an underscore since it's private, and can be a top-level function rather than a classmethod